server/configs/application.properties

```diff
@@ -0,0 +1,15 @@
+server.port=8081
+
+labkey.dataSourceName[0]=jdbc/labkeyDataSource
```

Comment:
Let's refine these property names. I propose a prefix like "labkeyDatabase" or "labkeyDataSource" instead of "labkey". Otherwise, it's hard to tell what you're supposed to be configuring from just the .properties file.